srcpkgs/lxqt-qt5plugin/template

```diff
@@ -1,13 +1,14 @@
 # Template file for 'lxqt-qt5plugin'
 pkgname=lxqt-qt5plugin
-version=1.4.1
-revision=2
+version=2.1.0
```

Comment:
this version is not for qt5: https://github.com/lxqt/lxqt-qtplugin/releases/tag/2.0.0